### PR TITLE
Sort the channels in the sidebar

### PR DESCRIPTION
**Pull request by gregkare**
Refs #45 

@galfert Would you add a test for something like this?

### CODE REVIEW

File: app/models/space.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ export default Ember.Object.extend({
 
   sockethubPersonId: function() {
     return `irc://${this.get('ircServer.nickname')}@${this.get('ircServer.hostname')}`;
-  }.property('ircServer.hostname', 'ircServer.nickname')
+  }.property('ircServer.hostname', 'ircServer.nickname'),
+
+  channelSorting: ['name'],
+  sortedChannels: Ember.computed.sort('channels', 'channelSorting')
```
Comment (skddc) [inline app/models/space.js]: Any reason this has to be a seperate variable here? It's actual making the same line longer than without one.

**Comment by skddc** [comment]
thumbs-up-cool2

File: app/models/space.js (hunk 1 above)
Comment (gregkare) [inline app/models/space.js]: Unless I'm missing something the second argument has to be an attribute as a string or a function (http://emberjs.com/api/classes/Ember.computed.html#method_sort)

Comment (skddc) [inline app/models/space.js]: Ah, didn't know that. Thanks.

**Comment by galfert** [comment]
Nice one.

I added a test for the channel sorting.

**Comment by gregkare** [comment]
Thanks for the test!

**Pull request merged**